prow/pubsub/reporter/reporter.go

```diff
@@ -43,12 +43,14 @@ const (
 
 // ReportMessage is a message structure used to pass a prowjob status to Pub/Sub topic.s
 type ReportMessage struct {
-	Project string               `json:"project"`
-	Topic   string               `json:"topic"`
-	RunID   string               `json:"runid"`
-	Status  prowapi.ProwJobState `json:"status"`
-	URL     string               `json:"url"`
-	GCSPath string               `json:"gcs_path"`
+	Project  string               `json:"project"`
+	Topic    string               `json:"topic"`
+	RunID    string               `json:"runid"`
+	Status   prowapi.ProwJobState `json:"status"`
+	URL      string               `json:"url"`
+	GCSPath  string               `json:"gcs_path"`
+	PRNumber int                  `json:"prnumber"`
+	JobType  prowapi.ProwJobType  `json:"job_type"`
```

Comment:
Should we include the job name and the PR SHA too?
It might be most appropriate to include a `*prowapi.Refs` field instead of the `PRNumber` field.

Reply:
And maybe a list of refs for extra_refs

Reply:
added the refs as well